systems/chromebook_oak/readme.md

```diff
@@ -45,6 +45,7 @@
   - with v5.10 it works fine (console and xorg) with https://github.com/hexdump0815/imagebuilder/blob/main/systems/chromebook_oak/extra-files/usr/lib/systemd/system-sleep/mrvl-reload - so all images starting with version 210724-02 should be fine
   - with v5.11+ it is broken as the display does not come back after resume - see todo
   - with v5.18 it is working again, only display sleep is broken and being worked around by special power management settings in the image
+  - with v6.1.0 it is working 
```

Comment:
is the display blanking really working for you with v6.1? - when i tested it last with my elm it still did not work ... currently there is a workaround in place in the settings -> power manager which simply reduces the brightness to only 1% after two minutes (which looks nearly like a proper display sleep state) but does not put the display into proper sleep state as - at least for me - it did not come back from there even with v6.1 ... the test is to enable the regular display blanking options in the power manager display settings and verify that the display comes back properly from a real display sleep

Reply:
When I had last looked at it, things were still working. I will test again later today and confirm.